snap/squashfs/squashfs.go

```diff
@@ -304,15 +304,19 @@ func (s *Snap) Build(buildDir string) error {
 	}
 
 	return osutil.ChDir(buildDir, func() error {
-		return exec.Command(
+		output, err := exec.Command(
 			"mksquashfs",
 			".", fullSnapPath,
 			"-noappend",
 			"-comp", "xz",
 			"-no-xattrs",
 			"-no-fragments",
 			"-no-progress",
-		).Run()
+		).CombinedOutput()
+		if err == nil {
```

Comment:
Hm, why not the usual pattern of `if err != nil {` here? Is there an advantage?

Reply:
I think the first iteration had more code below this check, so the "exit early" rule won